tests/base/boring_model.py

```diff
@@ -117,15 +126,26 @@ def test_epoch_end(self, outputs) -> None:
         torch.stack([x["y"] for x in outputs]).mean()
 
     def configure_optimizers(self):
-        optimizer = torch.optim.SGD(self.layer.parameters(), lr=0.1)
+        optimizer = getattr(torch.optim, self.optimizer_name)(self.layer.parameters(), lr=self.learning_rate)
         lr_scheduler = torch.optim.lr_scheduler.StepLR(optimizer, step_size=1)
         return [optimizer], [lr_scheduler]
 
     def train_dataloader(self):
-        return torch.utils.data.DataLoader(RandomDataset(32, 64))
+        return torch.utils.data.DataLoader(RandomDataset(32, 64), batch_size=self.batch_size)
 
     def val_dataloader(self):
-        return torch.utils.data.DataLoader(RandomDataset(32, 64))
+        return torch.utils.data.DataLoader(RandomDataset(32, 64), batch_size=self.batch_size)
 
     def test_dataloader(self):
-        return torch.utils.data.DataLoader(RandomDataset(32, 64))
+        return torch.utils.data.DataLoader(RandomDataset(32, 64), batch_size=self.batch_size)
+
+    def get_default_hparams(continue_training: bool = False, hpc_exp_number: int = 0) -> dict:
```

Comment:
this is out f the scope of this simple model, any way you request some arguments which are not later used